lerna.json

```diff
@@ -3,5 +3,6 @@
   "packages": [
```

Comment:
I think you need to update the lerna version here too, if you're going to upgrade lerna.